Simulation/read_results.m

```diff
@@ -0,0 +1,7 @@
+function var = read_results(file_name)
+  filename = [file_name ".mat"] ;%adds .mat to the file's name
+  fid = fopen (filename, "r");%creates a file of name filename. sets file to writable
+  fputs(fid, var);%put the variable into the file
+  fclose (fid); %close the file for good house keeping
+  var = 
```

Comment:
This looks like an incomplete line, unless I am unfamiliar with Octave's syntax?

Reply:
This file is not needed. Deleting it.